client/my-sites/themes/theme-showcase.jsx

```diff
@@ -388,12 +389,20 @@ class ThemeShowcase extends Component {
 			<div className="theme-showcase__all-themes">
 				<ThemesSelection { ...themesSelectionProps }>
 					{ showCollections && (
-						<ThemeCollectionsLayout
-							getOptions={ this.getThemeOptions }
-							getScreenshotUrl={ this.getScreenshotUrl }
-							getActionLabel={ this.getActionLabel }
-							onSeeAll={ this.onCollectionSeeAll }
-						/>
+						<>
+							<ShowcaseThemeCollection
+								{ ...THEME_COLLECTIONS.marketplace }
+								getOptions={ this.getThemeOptions }
+								getScreenshotUrl={ this.getScreenshotUrl }
+								getActionLabel={ this.getActionLabel }
+								onSeeAll={ () =>
+									this.onCollectionSeeAll( {
+										tier: THEME_COLLECTIONS.marketplace.query.tier,
```

Comment:
Could this just be `tier` and `filter` rather than hardcoding the marketplace collection?

Reply:
oh wait no, because `tier` at least will be `''`

Reply:
Yeah, we need the actual data from the Collection Definition (since we want to see all the themes in that specific collection) and not the current filter/tier selected by the user.